crates/cli/Cargo.toml

```diff
@@ -6,8 +6,8 @@ authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
 [dependencies]
 holochain_core_types = { path = "../core_types" }
 holochain_core = { path = "../core" }
-holochain_common = { path = "../common" }
-holochain_conductor_api = { path = "../conductor_api" }
+holochain_common = { path = "../../common" }
+holochain_conductor_api = { path = "../conductor_lib" }
```

Comment:
We should also change the crate name to `holochain_conductor_lib`.

Reply:
That is a good idea, I will change that right now